lexxpluss_tools/utils/make_meter_grid.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+echo '<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<gpx xmlns="http://www.topografix.com/GPX/1/1" creator="" version="1.1"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://www.topografix.com/GPX/1/1 ">http://www.topografix.com/GPX/1/1/gpx.xsd">
+<trk><name>Grid lines</name>'
+thr=0.000898315
+step=0.000008983
```

Comment:
make_centi_meter_grid.shのstepと比較して10倍のstepになっている。
centi_meterとmeterの差なので、100倍になるのかと思いました。
緯度方向に10倍、経度方向に10倍なので100倍という考え方でしょうか?

It is 10 times the step compared to the step in make_centi_meter_grid.sh.
I thought it was the difference between centi_meter and meter, so it would be 100x.
Is the idea that it is 10x in the direction of latitude and 10x in the direction of longitude, so 100x?

Reply:
centi meterという名称ですが、10cmのグリッドを作っているので10倍です。sub meterの方がよかったのかもしれないです。